98_exercises.Rmd

```diff
@@ -291,24 +291,87 @@ See the OMA book for examples.
 
 ### MultiAssayExperiment (MAE) data container
 
-1. Create TreeSE data containers from individual CSV files.
-2. Combine TreeSE into MAE.
-3. Check that each individual experiment of MAE equals corresponding TreeSE.
-4. Take a subset of MAE (e.g., 10 first samples), and observe the subsetted MAE.
-
-Useful functions: DataFrame, TreeSummarizedExperiment, matrix, rownames, colnames, MultiAssayExperiment, ExperimentList, SimpleList
-
+The following exercise walks you through disassembling a MAE object in
+order to store it as multiple separate csv files.
+
+1. Import the microbiomeDataSets package
```

Comment:
Note: 

HintikkaXOData is a special case that is available both in the mia package as well as through microbiomeDataSets. We can do either - - 

without download over the web:

```
data("HintikkaXOData", package="mia")
```

or 


with download over the web:

```
mae <- microbiomeDataSets::HintikkaXOData()
```

I would prefer the first approach since relying on network is not always good.